nautobot/ipam/models.py

```diff
@@ -888,6 +884,11 @@ class IPAddress(PrimaryModel):
         help_text="IPv4 or IPv6 host address",
     )
     mask_length = models.IntegerField(null=False, db_index=True, help_text="Length of the network mask, in bits.")
+    type = models.CharField(
```

Comment:
Doesn't this need to be added to ipam/forms.py?

How about `ipaddress.html`?

How about `ipam/tests/test_views.py`?